core/session.rb

```diff
@@ -55,6 +55,7 @@ class Session
   attr_accessor :productVersion
   attr_accessor :keyFile
   attr_accessor :keep_template
+  attr_accessor :list
```

Comment:
This method only adds the methods to access instance properties. You must also initialize the property in the initialize method.